wfe2/verify.go

```diff
@@ -600,6 +600,10 @@ func (wfe *WebFrontEndImpl) validPOSTAsGETForAccount(
 	if string(body) != "" {
 		return nil, probs.Malformed("POST-as-GET requests must have an empty payload")
 	}
+	// To make log analysis easier we choose to elevate the pseudo ACME HTTP
+	// method "POST-as-GET" to the logEvent's Method, replacing the
+	// http.MethodPost value.
+	logEvent.Method = "POST-as-GET"
```

Comment:
"POST-as-GET" is more bytes than the "PAG" value suggested in https://github.com/letsencrypt/boulder/issues/4389 but I'm hesitant to choose something less obvious when we're doing something pretty strange here in the grander scheme of HTTP.